sharding/nodesCoordinator/indexHashedNodesCoordinator.go

```diff
@@ -226,10 +229,46 @@ func checkArguments(args ArgNodesCoordinator) error {
 	if args.NumStoredEpochs < minStoredEpochs {
 		return ErrInvalidNumberOfStoredEpochs
 	}
+	if check.IfNilReflect(args.NodesConfigCache) {
```

Comment:
Please add the definition of `IsInterfaceNil() bool` in the Cacher definition and use here `check.IfNil()`

Reply:
done